packages/terra-toolkit-docs/src/terra-dev-site/tool/jest-config-terra/UpgradeGuide.tool.mdx

```diff
@@ -0,0 +1,8 @@
+# Jest Config Terra Upgrade Guide
+
+## Changes from 1.0 to 2.0
+
+`jest-config-terra` now supports Jest 27. Consumers will need to make the following updates in order to consume `jest-config-terra@2.0.0`:
```

Comment:
"Support" implies addition. However Jest 26 support has been dropped so I would change it to:
```suggestion
`jest-config-terra` now uses Jest 27. Consumers will need to make the following updates in order to consume `jest-config-terra@2.0.0`:
```

Reply:
[c1c18e8](https://github.com/cerner/terra-toolkit/pull/829/commits/c1c18e8b5f9ff32f18075de608e82ecf7374279c)